dls-controls/malcolmjs
# Feature/link theming

## Description

Links now respect theme colors mostly (their color during creation is still manually defined, but this definition has moved from index.css to link.factory.js)

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/gui/PANDA/layout
- make a new link and check its color during creation
- mouse over an existing link and click on it to select it, checking its color/style.
